### PR TITLE
Create xdebug addon

### PR DESCRIPTION
**Pull request by froboy**
A very simple addon to turn xdebug on or off (that mirrors [DDEV's function](https://ddev.readthedocs.io/en/latest/users/debugging-profiling/step-debugging/))

### CODE REVIEW

**Review by shelane — DISMISSED** [review]
Looks very similar to a custom command that I have.

**Comment by froboy** [comment]
@shelane thanks. Yea... it's been hanging out for a while and I figure it'd be nice to share. I could set no option to toggle the state, but figure doing nothing is always... non-destructive.

**Comment by shelane** [comment]
Please create in your PR an update to the readme file with this included in alphabetical order in the list.

**Review by shelane — CHANGES_REQUESTED** [review]
Readme needs update

**Review by shelane — APPROVED** [review]

File: xdebug/xdebug [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    on)
+        fin config set --env=local XDEBUG_ENABLED=1
+        fin up
+        ;;
+    off)
+        fin config set --env=local XDEBUG_ENABLED=0
+        fin up
+        ;;
+    *)
+        echo "Please tell me 'on' or 'off'."
```
Comment (shelane) [inline xdebug/xdebug]: Sorry, just one more nit-picky little thing. For consistency of verbiage with other commands, can you change this to:

`"Usage: fin xdebug <on|off>"`

and add `exit 1` on the next line?

**Review by shelane — CHANGES_REQUESTED** [review]
See inline comment

**Pull request merged**